packages/builders/src/interactions/modals/Modal.ts

```diff
@@ -82,20 +99,30 @@ export class ModalBuilder implements JSONEncodable<APIModalInteractionResponseCa
 	 *
 	 * @param components - The components to set
 	 */
-	public setComponents(...components: RestOrArray<ActionRowBuilder<ModalActionRowComponentBuilder>>) {
-		this.components.splice(0, this.components.length, ...normalizeArray(components));
+	public setActionRows(...components: RestOrArray<ActionRowBuilder>) {
```

Comment:
Should this have a `spliceActionRows` method, alike `EmbedBuilder#spliceFields()`?